src/game/character/skeleton.gd

```diff
@@ -32,8 +32,12 @@ var nodeStructure: Dictionary = {
 
 # Apply config from appearance's variables
 func applyAppearance(outfit: Dictionary, colors: Dictionary) -> void:
-	_applyOutfit(outfit)
-	_applyColors(colors)
+	if outfit.empty() or colors.empty():
+		assert(not outfit.empty(), "Missing outfit data")
+		assert(not colors.empty(), "Missing colors data")
+	else:
```

Comment:
you don't need to put the asserts inside an "if" statement, that will result in an empty conditional for the non-debug build.
the assert should be  just performed before _applyOutfit and _applyColor.
And also, keep in mind, the assert is a debugging tool. I.e., it helps us make sure, that we don't ever call this function with empty outfit or colors dictionaries, because if we do, that means that somewhere we don't handle a situation properly. So, please make sure that when the closet is empty and we press "Load", it actually checks whether there is something to load, and if it's empty, that should be handled at that point in the code.